application/index.html

```diff
@@ -15,6 +15,7 @@
         <meta name="apple-mobile-web-app-title" content="Lively Docs">
         <link rel="stylesheet" href="/css/app.css">
         <script src="/js/vendor-header.js"></script>
+        <script type="text/javascript" src="https://js.stripe.com/v2/"></script>
```

Comment:
I don't think we should include this very specialized script unless it's used for one of the APIs in the config.